seatunnel-dist/release-docs/LICENSE

```diff
@@ -896,6 +896,7 @@ The text of each license is the standard Apache 2.0 license.
      (The Apache Software License, Version 2.0) transport-netty4 (org.elasticsearch.plugin:transport-netty4-client:7.5.1 - https://github.com/elastic/elasticsearch)
      (The Apache Software License, Version 2.0) x-content (org.elasticsearch:elasticsearch-x-content:6.3.1 - https://github.com/elastic/elasticsearch)
      (The Apache Software License, Version 2.0) zookeeper (org.apache.zookeeper:zookeeper:3.4.10 - no url defined)
+     (The Apache Software License, Version 2.0) spark-excel (com.crealytics:spark-excel_2.11:0.11.1 - https://github.com/crealytics/spark-excel)
```

Comment:
Other licenses also need to be added here

Reply:
I introduced the spark-excel tripartite package. In addition to listing the license of spark-excel, I also need to list the licesne of the library it depends on, right?

Reply:
Seem like you should add other license in this file. like 
```
poi-4.0.0.jar
poi-ooxml-4.0.0.jar
poi-ooxml-schemas-4.0.0.jar
```
They are new jar you add on known-dependcies.txt, so you should add license link on this file, you can copy example from already exist license.